### PR TITLE
2018_Users_Mtg_Tools_for_OMERO_Users

### PR DESCRIPTION
**Pull request by will-moore**
Work in progress...

 - Talk staged at http://will-moore.github.io/presentations/2018/Users-Meeting/Talks/Tools-for-OMERO-Users/slides/
 - Metadata Analysis: http://will-moore.github.io/presentations/2018/Users-Meeting/Workshops/Metadata-Analysis/slides/
 - Metadata Handling: http://will-moore.github.io/presentations/2018/Users-Meeting/Workshops/Metadata-Handling/slides/

### CODE REVIEW

File: 2018/Users-Meeting/Talks/Tools-for-OMERO-Users/slides/index.html [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+                    <h2>"Tools": OMERO apps and CLI clients</h2>
+
+                    <ul>
+                        <li>OMERO.scripts</li>
+                        <li>Web Apps:
+                            <ul>
+                                <li>figure</li>
+                                <li>iviewer</li>
+                                <li>FPBioImage</li>
+                                <li>webtagging</li>
```
Comment (jburel) [inline 2018/Users-Meeting/Talks/Tools-for-OMERO-Users/slides/index.html]: need to differentiate between the ones we maintain and others.
webtagging is not under our controls. (not release for 1 year)
there is also omero-forms

File: 2018/Users-Meeting/Talks/Tools-for-OMERO-Users/slides/index.html [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+                                <li>parade</li>
+                            </ul>
+                        </li>
+                        <li>CLI:
+                            <ul>
+                                <li>render</li>
```
Comment (jburel) [inline 2018/Users-Meeting/Talks/Tools-for-OMERO-Users/slides/index.html]: in progress metadata

File: 2018/Users-Meeting/Talks/Tools-for-OMERO-Users/slides/index.html [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+                    <h2>Testing of decoupled clients</h2>
+
+                    <ul>
+                        <li>
+                            Use <a target="_blank" href="https://github.com/openmicroscopy/omero-test-infra">
+                                omero-test-infra
+                            </a>
+                        </li>
+                        <li>Docker-composed OMERO server for testing</li>
+                        <li>Testing scripts run by Travis CI on every github PR</li>
```
Comment (jburel) [inline 2018/Users-Meeting/Talks/Tools-for-OMERO-Users/slides/index.html]: GitHub

File: 2018/Users-Meeting/Workshops/Metadata-Analysis/slides/index.html [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+                <section>
+                    <h2>Client-side vs Server-side analysis</h2>
+                    <img src="images/omero-extension-points.png"/>
+                </section>
+
+
+                <section>
+                    <h2>Sample data: idr0021</h2>
+                    <iframe style="width: 1000px; max-width: 1000px; height: 600px; max-height: 600px"
+                        src="http://idr.openmicroscopy.org/"></iframe>
```
Comment (jburel) [inline 2018/Users-Meeting/Workshops/Metadata-Analysis/slides/index.html]: https

File: 2018/Users-Meeting/Workshops/Metadata-Analysis/slides/index.html [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+                </section>
+
+
+                <section>
+                    <h2>
+                        Subdiffraction imaging of centrosomes reveals higher-order organizational
+                        features of pericentriolar material
+                    </h2>
+                    <p>Steffen Lawo, Monica Hasegan, Gagan D. Gupta & Laurence Pelletier</p>
+                    <p><a target="_blank" href=https://www.nature.com/articles/ncb2591">
```
Comment (jburel) [inline 2018/Users-Meeting/Workshops/Metadata-Analysis/slides/index.html]: missing ``"`` after href

File: 2018/Users-Meeting/Workshops/Metadata-Handling/slides/index.html [hunk 6]
```diff
@@ -157,12 +160,14 @@ <h2>OMERO.parade</h2>
...
     '["Parade", "omero_parade/init.js.html", "omero_parade"]'</code></pre>
 
-                    <small><a target="_blank" href="https://pypi.org/project/omero-parade/">OMERO.parade</a></small>
                 </section>
 
 
                 <section>
-                    <h2>OMERO.figure usage of Metadata</h2>
+                    <h2>
+                        <a target="_blank" href="http://www.openmicroscopy.org/omero/figure/">
```
Comment (jburel) [inline 2018/Users-Meeting/Workshops/Metadata-Handling/slides/index.html]: https

**Comment by sbesson** [comment]
Once @jburel is happy with the last set of changes, we should be ready to get this in a first round of integration of the presentations

**Comment by sbesson** [comment]
Looking at the files, what is the rationale for the extra `slides/` spacer? Is other binary content expected to be added next to it?

**Comment by jburel** [comment]
@will-moore do you want to add the link to access notebook? (Metadata-analysis)
https://idr-analysis.openmicroscopy.org/training/hub/tmplogin

**Comment by jburel** [comment]
@sbesson yes we will have the walkthroughs (pdf)

**Comment by jburel** [comment]
* Maybe add version of iviewer and figure (version is available for others)

**Comment by jburel** [comment]
OMERO.player is from your personal repo. so we should not list it.

**Comment by will-moore** [comment]
I want to talk about OMERO.player as an example of a prototype app that is still a prototype.
The point I want to make is that one advantage of separate web apps is that you can test things out in prototypes that are not necessarily ready to be used in production.
The https://github.com/will-moore/omero-player page clearly states "NB: This is a personal prototype app and is not supported by the OME team."

**Comment by will-moore** [comment]
```Maybe add version of iviewer and figure (version is available for others)```
Any suggestion as to where? I had to remove this for some slides to save space.
I could add it in the title e.g. "OMERO.figure 4.0.0" ?

**Comment by will-moore** [comment]
I've not put the outreach server URL in the workshop slides either, and we didn't put any links like this in Cambridge presentations, so I think I will leave it out for now.

**Comment by jburel** [comment]
cf. personal page. We discussed that during deployment workshop. It is fine to indicate that it is a prototype but in that case we need to migrate it to gitlab under incubator
https://gitlab.com/projects/new?namespace_id=2379008

**Comment by jburel** [comment]
version as indicated is good

**Comment by will-moore** [comment]
https://gitlab.com/projects/new?namespace_id=2379008 is 404 for me, logged in as will23.

**Comment by jburel** [comment]
I have not transferred anything 
If you go to https://gitlab.com then New project then select Import Project tab 
you should be able to select omero-player from your repo

**Comment by will-moore** [comment]
OK, I'll update link to https://gitlab.com/will23/omero-player

**Comment by jburel** [comment]
it should go under openmicroscopy/incubator
i.e. https://gitlab.com/openmicroscopy/incubator/omero-player

**Comment by will-moore** [comment]
I'm in "Transfer Project" > "Select a new namespace" but I only see "Will Moore". Should I expect to see "openmicroscopy" there?

**Comment by jburel** [comment]
Done 
https://gitlab.com/openmicroscopy/incubator/omero-player

**Comment by jburel** [comment]
Not related to this PR but since we have now moved omero-player. Its readme will need to be updated i.e. remove the ``NB personal page``

**Comment by jburel** [comment]
Looks good now thanks @will-moore 
merging

**Pull request merged**